src/main/java/com/google/sps/servlets/utils/CommentAnalysis.java

```diff
@@ -52,9 +52,9 @@ public CommentAnalysis(LanguageServiceClient languageService) {
    *
    * @return a Statistics object that contains required values to display
    */
-  public Statistics computeOverallStats(CommentThreadListResponse youtubeResponse) {
+  public Statistics computeOverallStats(List<CommentThread> youtubeResponse) {
     List<Double> scoreValues =
-        youtubeResponse.getItems().stream()
+        youtubeResponse.stream()
```

Comment:
To make this operation faster, you can consider executing it in parallel. This will be especially important if we end up processing many 100s of comments.

You can do this by using `parallelStream()` instead of `stream()`. This will ensure that the sentiment analysis API will be invoked with multiple comments concurrently instead of serially.

Reply:
I think Xin might have already done this in one of her PR's. CommentAnalysis is more her area, I only had to modify it so that the changes made to make this pr possible wouldn't break it.